src/displayapp/screens/settings/SettingSetTime.cpp

```diff
@@ -67,7 +67,7 @@ SettingSetTime::SettingSetTime(Pinetime::Applications::DisplayApp* app,
 
   btnSetTime = lv_btn_create(lv_scr_act(), nullptr);
   btnSetTime->user_data = this;
-  lv_obj_set_size(btnSetTime, 120, 50);
+  lv_obj_set_size(btnSetTime, LV_HOR_RES_MAX - 30, 48);
```

Comment:
Please don't add unexplained changes. The button height should be 50, so you may update these accordingly. Please read the Commits section of the maintainer's guide to see the requirements for commits. These will soon be added to the contributing guide #1570. https://github.com/InfiniTimeOrg/InfiniTime/blob/develop/doc/maintainer-guide.md#commits

Reply:
It should be 50, because 50 is standard on many screens, and I think it being different here is a mistake. I agree there should be some more distance between the buttons, however it should be addressed some other way, perhaps by adjusting the counter size or position, and ideally in a separate PR.

Reply:
Ah, Ok. Should I make the button height for setDate 50 also?

Reply:
You could. You don't have to.